README.md

```diff
@@ -203,422 +203,566 @@ email=jchaloup@redhat.com
+        "test_guidance": "<comma-separated-list-of-test-suites-to-run>",
+        "message-content": "",
+        "status": "<success/failure>"
+  }
 }
 ````
 
 #### org.centos.prod.ci.pipeline.package.queued
 
 ````
 { 
-  u'i': 1,
-  u'msg': { u'build_url': u'<full-jenkins-url-to-build>',
-            u'package_url': u'<full-url-to-package>',
-            u'build_id': <Jenkins-build-id-number>,
-            u'ref': u'fedora/rawhide/${basearch}/atomic-host',
-      	u'rev': u'<sha of the commit from dist-git>', 
-      	u'namespace': u'rpms', 
-      	u'repo': u'php-simplepie', 
-            u'status': u'<success/aborted>',
-            u'test_guidance': u'<comma-separated-list-of-test-suites-to-run>'},
-  u'msg_id': u'2015-4b5aae66-b713-4c22-bb4a-1277d4402375',
-  u'timestamp': 1379638157.759283,
-  u'topic': u'org.centos.prod.ci.pipeline.package.queued',
-  u'username': u'fedora-atomic'
+  "i": 1,
+  "timestamp": 1496382015, 
+  "msg_id": "2017-f2b46ca1-1b07-4539-8d86-f7d3cb1b5934", 
+  "crypto": "x509", 
+  "topic": "org.centos.prod.ci.pipeline.package.queued",   
+  'msg': { 
+        "CI_TYPE": "custom", 
+        "CI_NAME": "ci-pipeline-rpmbuild",
+        "topic": "org.centos.prod.ci.pipeline.package.queued",
+        "build_id": "79", 
+        "build_url": "https://ci.centos.org/job/ci-pipeline-rpmbuild/79/",
+        "branch": "rawhide",
+        "ref": "fedora/rawhide/x86_64/atomic-host",
+        "rev": "0546fa18041a8ca223f4f441dc1868fc81ddce0f", 
+        "repo": "perl", 
+        "namespace": "rpms", 
+        "username": "fedora-atomic", 
+        "test_guidance": "<comma-separated-list-of-test-suites-to-run>",
+        "message-content": "",
+        "status": "<success/failure>"
+  }
 }
 ````
 
 #### org.centos.prod.ci.pipeline.package.running
 
 ````
 { 
-  u'i': 1,
-  u'msg': { u'build_url': u'<full-jenkins-url-to-build>',
-            u'package_url': u'<full-url-to-package>',
-            u'build_id': <Jenkins-build-id-number>,
-            u'ref': u'fedora/rawhide/${basearch}/atomic-host',
-      	u'rev': u'<sha of the commit from dist-git>', 
-      	u'namespace': u'rpms', 
-      	u'repo': u'php-simplepie', 
-            u'status': u'<success/aborted>',
-            u'test_guidance': u'<comma-separated-list-of-test-suites-to-run>'},
-  u'msg_id': u'2015-4b5aae66-b713-4c22-bb4a-1277d4402375',
-  u'timestamp': 1379638157.759283,
-  u'topic': u'org.centos.prod.ci.pipeline.package.running',
-  u'username': u'fedora-atomic'
+  "i": 1,
+  "timestamp": 1496382015, 
+  "msg_id": "2017-f2b46ca1-1b07-4539-8d86-f7d3cb1b5934", 
+  "crypto": "x509", 
+  "topic": "org.centos.prod.ci.pipeline.package.running",   
+  'msg': { 
+        "CI_TYPE": "custom", 
+        "CI_NAME": "ci-pipeline-rpmbuild",
+        "topic": "org.centos.prod.ci.pipeline.package.running",
+        "build_id": "79", 
+        "build_url": "https://ci.centos.org/job/ci-pipeline-rpmbuild/79/",
+        "package_url": "<full-url-to-package>",
```

Comment:
These messages kick off when the rpmbuild job starts, I don't think we have package_url until the .complete message